test/asynchronous/test_auth.py

```diff
@@ -169,7 +169,7 @@ async def test_gssapi_simple(self):
         client = AsyncMongoClient(mech_uri)
         await client[GSSAPI_DB].collection.find_one()
 
-        set_name = await client.admin.command(HelloCompat.LEGACY_CMD).get("setName")
```

Comment:
We can probably replace these setName lines with `client_context.replica_set_name`.